extension/scripts/features/abroad-people-filter/ttPeopleFilter.js

```diff
@@ -261,7 +261,7 @@
 
 		function show() {
 			row.classList.remove("hidden");
-			delete row.dataset.hideReason;

```

Comment:
Why would you change this?

Reply:
Because `delete` takes 3ms while this does that fast.